src/plot_api/plot_api.js

```diff
@@ -2593,10 +2493,11 @@ function makePlotFramework(gd) {
     var gd3 = d3.select(gd),
         fullLayout = gd._fullLayout;
 
-    /*
-     * TODO - find a better place for 3D to initialize axes
-     */
-    if(fullLayout._hasGL3D) Plotly.Gl3dLayout.initAxes(gd);
+    // TODO - find a better place for 3D to initialize axes
+    var plotRegistry = plots.subplotsRegistry;
+    if(fullLayout._hasGL3D && plotRegistry.gl3d) {
```

Comment:
Same thing here - do we need to check both or just let it fail and throw an error if `plotRegistry.gl3d` is undefined?